<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>611</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="inputLabel">
       <property name="text">
        <string>Please provide your text here:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="inputTxt"/>
     </item>
     <item>
      <widget class="QLabel" name="resultLabel">
       <property name="text">
        <string>Here is your result:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="resultTxt"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>611</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="checkB">
       <property name="text">
        <string>check</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetB">
       <property name="text">
        <string>reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeB">
       <property name="text">
        <string>close</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>635</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMy_Language_Tool">
    <property name="title">
     <string>My Language Tool</string>
    </property>
   </widget>
   <addaction name="menuMy_Language_Tool"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
